libs/api/core/data-access/src/lib/api-core-data-access.service.ts

```diff
@@ -229,21 +227,6 @@ export class ApiCoreDataAccessService extends PrismaClient implements OnModuleIn
     return `app-${index}-${environment}`
   }
 
-  async getSolanaConnection(environment: string, index: number): Promise<Solana> {
```

Comment:
A nice result of this cleanup is that we remove some app specific logic from `ApiCoreDataAccessService`.